src/main/java/seedu/partyplanet/logic/commands/EditToRemoveTagCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.partyplanet.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.partyplanet.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.partyplanet.logic.commands.exceptions.CommandException;
+import seedu.partyplanet.model.Model;
+import seedu.partyplanet.model.person.Person;
+import seedu.partyplanet.model.tag.Tag;
+
+public class EditToRemoveTagCommand extends EditCommand {
+
+    public static final String MESSAGE_REMOVE_TAG_SUCCESS = "Removed tag from: %1$s";
+    public static final String MESSAGE_TAG_NOT_REMOVED = "At least one tag to remove must be provided.";
+
+    private final Tag targetTag;
+
+    private final List<Person> editedPersons = new ArrayList<Person>();
+
+    /**
+     * Creates an EditToRemoveTagCommand to edit the {@code Person} with specified {@code Tag}
+     */
+    public EditToRemoveTagCommand(Tag targetTag) {
+        this.targetTag = targetTag;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Person> lastShownList = model.getFilteredPersonList();
+
+        for (Person person: lastShownList) {
+            removeTagFromPerson(model, person);
+        }
+        model.addState();
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+
+        return new CommandResult(String.format(MESSAGE_REMOVE_TAG_SUCCESS,
+                editedPersons.isEmpty() ? "" : displayPersons(editedPersons)));
```

Comment:
Would it be better for us to display another message if the tag specified is not found in the listed persons?

Currently it is `"Removed tag from: "`. Maybe we can change it to 
eg. `"This tag does not exist in the persons listed. No tags were removed."`

Reply:
If implementing, can consider allowing `removeTagFromPerson` to return a boolean reflecting the tag removal success, just like the `boolean isEdited = tags.remove(targetTag);` line you used in line 49.

Reply:
On a related note, since `model.addState()` should not be called if no tag removal operations were performed, I think doing the check would be a good idea. Ties in with the same idea.

Reply:
Adopted in latest commit.